QModManager/HarmonyPatches/DisableFeedbackButton.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace QModManager.HarmonyPatches.DisableFeedbackButton
+{
+    using Harmony;
+    using UnityEngine;
+    using UnityEngine.UI;
+
+    [HarmonyPatch(typeof(IngameMenu), nameof(IngameMenu.Start))]
+    internal static class IngameMenu_Start
+    {
+        [HarmonyPostfix]
+        internal static void Postfix(IngameMenu __instance)
+        {
+            __instance.feedbackButton.interactable = false;
+            Transform transform = __instance.transform.Find("Main/ButtonLayout/ButtonFeedback");
+            if (transform != null) transform.GetComponent<Button>().interactable = false;
```

Comment:
I'm still not convinced that QMM should be removing the feedback and error report features.

Reply:
I believe the removal of this will help in BZ most of all because the Game Develpment team wont have to figure out if the bug is caused by modding or is a base game bug they actually need to fix.